bessctl/cli.py

```diff
@@ -441,6 +430,52 @@ def restore_echoctl(self):
         except:
             pass
 
+    def go_interactive(self):
```

Comment:
So actually I went back on this, because if there's no TTY, it doesn't go into interactive mode -- so it *is* maybe! I changed the name as you suggested, ha.